docs/lib/predictions/fragments/native_common/getting-started/common.md

```diff
@@ -1,39 +1,37 @@
-The Predictions category for enables you to integrate machine learning in your application without any prior machine learning experience.  We support a number of use cases which include, but not limited to: translating text from one language to another, converting text to speech, text recognition from an image, entities recognition, labeling real world objects, interpretation of text, uploading images for automatic training, and (currently, iOS only) transcribing text.  These use cases are powered by Amazon services including: [Amazon Translate](https://docs.aws.amazon.com/translate/latest/dg/what-is.html), [Amazon Polly](https://docs.aws.amazon.com/polly/latest/dg/what-is.html), [Amazon Transcribe](https://docs.aws.amazon.com/transcribe/latest/dg/what-is-transcribe.html), [Amazon Rekognition](https://docs.aws.amazon.com/rekognition/latest/dg/what-is.html), [Amazon Textract](https://docs.aws.amazon.com/textract/latest/dg/what-is.html), and [Amazon Comprehend](https://docs.aws.amazon.com/comprehend/latest/dg/what-is.html).
+The Predictions category for enables you to integrate machine learning in your application without any prior machine learning experience.  We support a number of use cases which include, but not limited to: translating text from one language to another, converting text to speech, text recognition from an image, entities recognition, labeling real world objects, interpretation of text, and uploading images for automatic training. These use cases are powered by Amazon services including: [Amazon Translate](https://docs.aws.amazon.com/translate/latest/dg/what-is.html), [Amazon Polly](https://docs.aws.amazon.com/polly/latest/dg/what-is.html), [Amazon Transcribe](https://docs.aws.amazon.com/transcribe/latest/dg/what-is-transcribe.html), [Amazon Rekognition](https://docs.aws.amazon.com/rekognition/latest/dg/what-is.html), [Amazon Textract](https://docs.aws.amazon.com/textract/latest/dg/what-is.html), and [Amazon Comprehend](https://docs.aws.amazon.com/comprehend/latest/dg/what-is.html).
```

Comment:
Should we put the enabled use cases into an ordered or unordered list?

Reply:
I don't feel strongly here. I think it actually looks fine as a graf, and slightly prefer that as starting the document off with a list seems odd.

![CleanShot 2020-05-23 at 15 06 12](https://user-images.githubusercontent.com/19775/82738626-fa203200-9d06-11ea-9db8-ba222ac4eb15.png)